internal/utils/comparators/servicemonitor/comparator.go

```diff
@@ -4,72 +4,72 @@ import (
 	"reflect"
 
 	log "github.com/ViaQ/logerr/v2/log/static"
-	monitoringv1 "github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1"
 	"github.com/openshift/cluster-logging-operator/internal/utils"
+	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
 )
 
 // AreSame compares for equality and return true equal otherwise false
 func AreSame(current *monitoringv1.ServiceMonitor, desired *monitoringv1.ServiceMonitor) bool {
-	log.V(3).Info("Comparing Services current to desired", "current", current, "desired", desired)
+  log.V(3).Info("Comparing Services current to desired", "current", current, "desired", desired)
 
-	if !utils.AreMapsSame(current.ObjectMeta.Annotations, desired.ObjectMeta.Annotations) {
-		log.V(3).Info("ServiceMonitor  annotation change", "current name", current.Name)
-		return false
-	}
+  if !utils.AreMapsSame(current.ObjectMeta.Annotations, desired.ObjectMeta.Annotations) {
+    log.V(3).Info("ServiceMonitor  annotation change", "current name", current.Name)
+    return false
+  }
 
-	if !utils.AreMapsSame(current.ObjectMeta.Labels, desired.ObjectMeta.Labels) {
-		log.V(3).Info("ServiceMonitor label change", "current name", current.Name)
-		return false
-	}
+  if !utils.AreMapsSame(current.ObjectMeta.Labels, desired.ObjectMeta.Labels) {
+    log.V(3).Info("ServiceMonitor label change", "current name", current.Name)
+    return false
+  }
```

Comment:
is this change done after running `go fmt` ?  the Go standard is to use tabs

Reply:
same comment for other files

Reply:
hm, strange i run `make fmt`